.circleci/config.yml

```diff
@@ -67,9 +67,9 @@ jobs:
       - deploy:
           command: |
             if [ "${CIRCLE_BRANCH}" == "develop" ]; then
-            echo "$MAVEN_CENTRAL_SEC_RING" | base64 -d > $HOME/secring.gpg
-            gpg --import --batch $HOME/secring.gpg
-              ./gradlew publish -d -Psonatype.username=$SONATYPE_USERNAME -Psonatype.password=$SONATYPE_PASSWORD -Psigning.keyId=0E7A8B89 -Psigning.password=$MAVEN_CENTRAL_KEY_PASSPHRASE -Psigning.secretKeyRingFile=$HOME/secring.gpg -Porg.gradle.parallel=false
+              echo "$MAVEN_CENTRAL_SEC_RING" | base64 -d > $HOME/secring.gpg
+              gpg --import --batch $HOME/secring.gpg
+              ./gradlew publish -Psonatype.username=$SONATYPE_USERNAME -Psonatype.password=$SONATYPE_PASSWORD -Psigning.keyId=0E7A8B89 -Psigning.password='$MAVEN_CENTRAL_KEY_PASSPHRASE' -Psigning.secretKeyRingFile=$HOME/secring.gpg -Porg.gradle.parallel=false
```

Comment:
should go ahead and quote the username/password too, just in case they ever change.

Reply:
Also, shouldn't they be double quotes around the passphrase?

Reply:
I think that's fine, I only noticed in my editor it highlighted a $prefixed environment variable in a double quoted string, but not in a single quoted string